### PR TITLE
rework jasmine to ts

### PR DESCRIPTION
**Pull request by CrispusDH**
@cnishina , could you help me with `spec/custom/framework.js`. How to modified this file properly?
@heathkit , could you take a look at it if you have a time

### CODE REVIEW

File: lib/frameworks/jasmine.js [hunk 1]
```diff
@@ -1,125 +0,0 @@
...
-  require('./setupAfterEach').setup(runner, specs);
-
-  // Filter specs to run based on jasmineNodeOpts.grep and jasmineNodeOpts.invert.
-  jasmine.getEnv().specFilter = function(spec) {
-    var grepMatch = !jasmineNodeOpts ||
-        !jasmineNodeOpts.grep ||
-        spec.getFullName().match(new RegExp(jasmineNodeOpts.grep)) != null;
-    var invertGrep = !!(jasmineNodeOpts && jasmineNodeOpts.invertGrep);
-    if (grepMatch == invertGrep) {
-      spec.disable();
```
Comment (CrispusDH) [inline lib/frameworks/jasmine.js]: `jasmine.Spec` does not have such method

**Comment by CrispusDH** [comment]
Reference #5061

**Comment by CrispusDH** [comment]
@cnishina any updates?

**Comment by CrispusDH** [comment]
@cnishina I found the right path

File: spec/custom/framework.js [hunk 2]
```diff
@@ -4,6 +4,5 @@
  *
  * @param {Runner} runner The current Protractor Runner.
  * @param {Array} specs Array of Directory Path Strings.
- * @return {q.Promise} Promise resolved with the test results
  */
-exports.run = require('../../lib/frameworks/jasmine.js').run;
+exports.run = require('../../built/frameworks/jasmine').run;
```
Comment (cnishina) [inline spec/custom/framework.js]:
```suggestion
exports.run = require('../../built/lib/frameworks/jasmine').run;
```

The lib path still is required.

Comment (CrispusDH) [reply spec/custom/framework.js]: Hm, when I made `tsc` there is no `lib` folder inside the `built`. Also, I checked that with current path it works.

**Comment by CrispusDH** [comment]
@cnishina , both CIs were passed

File: spec/custom/framework.js (hunk 2 above)
Comment (cnishina) [reply spec/custom/framework.js]: Yup. I agree.

**Comment by CrispusDH** [comment]
create new PR #5189

**Pull request closed without merging**